src/lib/containers/works_for_you.tsx

```diff
@@ -40,6 +40,9 @@ interface State {
 }
 
 export class WorksForYou extends React.Component<Props, State> {
+  listView: ListView
+  currentScrollOffset: number
```

Comment:
These should probably be typed as optional, seeing as they’re not set until the first render.